# ~/ros2_ws/src/aruco_navigator/config/calibrated_camera_params_320x320.yaml
# (새로운 캘리브레이션 데이터로 수정된 내용)

aruco_detector: # <--- 아루코 감지 노드의 이름과 일치해야 합니다.
  ros__parameters:
    camera_matrix: [1079.1402620038577, 0.0, 164.2439825517726, 0.0, 4256.08491654803, 169.4840079247005, 0.0, 0.0, 1.0] # <--- 단일 리스트로 변환됨
    dist_coeffs: [76.0576412882301, -21369.94666804956, -0.41411720780555167, 0.339603916214559, -171.6341226812124]
    marker_size: 0.1 # <--- 당신의 실제 ArUco 마커 한 변의 크기 (미터 단위)를 입력하세요.
                     #      예: 10cm 마커라면 0.1, 5cm 마커라면 0.05